docuilib/src/components/ComponentPage.tsx

```diff
@@ -87,9 +93,15 @@ export default function ComponentPage({component}) {
   };
 
   return (
-    <div>
-      {buildHero()}
-      {buildTabs()}
-    </div>
+    <BrowserOnly>
```

Comment:
What's the BrowserOnly for? Does it fixes an issue?

Reply:
I've just double checked on private, it didn't fix. 
It did fix in public, but I guess it wasn't the same.

Im removing it